README.md

```diff
@@ -578,10 +578,20 @@ Options available here are shared with `additional_repositories`.
 
 ```yaml
 git:

```

Comment:
You documented a lot of new options here, but only "tags" is explicit tested.
Is the rest already somehow tested?

Reply:
Yes, supported and tested, just not documented for some reason.

Reply:
OK.

That the "key" types are validated just by the outer most type and no structure do not help us with this either =)

https://github.com/ESSS/jobs_done10/blob/master/src/jobs_done10/jobs_done_job.py#L73-L79

Reply:
Yep, thought the same. 😖

Reply:
Maybe add that to documentation in "readme".

Reply:
What do you mean?

Reply:
ah... didn't notice this was on readme, my bad.